extensions/machine-learning-services/src/packageManagement/packageManager.ts

```diff
@@ -78,16 +107,12 @@ export class PackageManager {
 				isCancelable: false,
 				operation: async op => {
 					try {
-						if (!(await utils.exists(this._pythonExecutable))) {
-							// Install python
-							//
-							await utils.createFolder(this._pythonInstallationLocation);
-							await this.jupyterInstallation.installPythonPackage(op, false, this._pythonInstallationLocation, this._outputChannel);
-						}
+						await utils.createFolder(utils.getPackagesFolderPath(this._rootFolder));
 
 						// Install required packages
 						//
 						await this.installRequiredPythonPackages();
+						await this.installRequiredRPackages(op);
 						op.updateStatus(azdata.TaskStatus.Succeeded);
```

Comment:
How long do these take? It might be worthwhile to look into kicking them off simultaneously for perf